timemachine/fe/single_topology.py

```diff
@@ -1061,6 +1221,23 @@ def verify_chiral_consistency_of_core(mol_a: Chem.Mol, mol_b: Chem.Mol, core: ND
     )
 
 
+def is_chirally_consistent(
+    mol_a,
+    mol_b,
+    core,
+    dummy_groups_ab: Dict[int, FrozenSet[int]],
+    dummy_groups_ba: Dict[int, FrozenSet[int]],
+    ff: Forcefield,
+) -> bool:
+    try:
+        verify_chiral_consistency(mol_a, mol_b, core, dummy_groups_ab, dummy_groups_ba, ff)
+        return True
+    # TODO: understand why AssertionError is raised
+    # (i.e. some dummy group assignments induce chiral inversions)
```

Comment:
Can you also confirm if for the cases where an inversion is happening, if it's happening over only core atoms (which should be forbidding by MCS) or if it's happening over a mix of core dummy atoms? I suspect the latter, esp if it's dummy group assignment dependent.

Reply:
Looking more closely, the example I'd found was `test_ring_matches_ring_only`, which sets `enforce_chiral=False`: https://github.com/proteneer/timemachine/blob/3a174e63d32f4813f2d6cf4e9de4a677e6ba7419/tests/test_atom_mapping.py#L889

I see 2 options:
1. Make `disallow_chiral_conversion` imply `enforce_chiral` (i.e. raise an error if `disallow_chiral_conversion` but not `enforce_chiral`)
2. Separate the checks for chiral consistency (i.e. consistent chiral volume restraints) and validity (i.e. no volume sign changes), make `disallow_chiral_conversion` only check for consistency but not validity

Reply:
Opted for (2) in https://github.com/proteneer/timemachine/pull/1356/commits/be96e1f64453d95b351c29d246f8ab4212de183e and https://github.com/proteneer/timemachine/pull/1356/commits/fad5e579496ff388682a9445dce5b7c9e26cf182, since it seemed cleanest to keep `disallow_chiral_conversion` and `enforce_chiral` independent.

Reply:
sgtm! And this explanation makes sense.